src/flexmock/_pytest_plugin.py

```diff
@@ -0,0 +1,26 @@
+# Hook flexmock into Pytest.
```

Comment:
Can you change this comment to a docstring?

Reply:
done